UI/css/ledgersmb.css

```diff
@@ -73,11 +73,15 @@ table.warning {
     border: 0;
 }
 
-#company_info, #version_info, #login_info {
+#version_info {
    font-weight: bold;
    font-size: 9pt;
 }
 
+#company_info, #login_info {
```

Comment:
It is duplicated in many places, see https://github.com/ledgersmb/LedgerSMB/search?utf8=%E2%9C%93&q=version_info&type=
This should be addressed in another PR